Packaging/OpenDingux/build-rg350.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+declare -r DIR="$(dirname "${BASH_SOURCE[0]}")"
+cd "$DIR"
+declare -r ABSDIR="$(pwd)"
+
+BUILDROOT="${BUILDROOT:-$HOME/buildroot-rg350-devilutionx}"
+
+set -x
+
+main() {
+	set -x
+	prepare_buildroot
+	build
+	package
+}
+
+prepare_buildroot() {
+	if [[ -d $BUILDROOT ]]; then
+		return
+	fi
+	git clone --depth=1 -b backport-glibc-host-fix https://github.com/glebm/RG350_buildroot.git "$BUILDROOT"
```

Comment:
Why not use https://github.com/tonyjih/RG350_buildroot It doesn't appear that your fork has any commits on top of it as is already lacking a bit behind?

Reply:
My PR to it was just merged today :)

Reply:
Done